packages/core/src/browser/style/variables-dark.useable.css

```diff
@@ -196,6 +196,7 @@ is not optimized for dense, information rich UIs.
   --theia-scrollbar-rail-color: transparent;
   --theia-scrollbar-active-thumb-color: hsla(0,0%,61%,.4);
   --theia-scrollbar-active-rail-color: transparent;
+  --theia-scrollbar-box-shadow-color: black;
```

Comment:
For the sake of consistency, can you provide the `hex` value instead like other colors.

Reply:
Also, we should probably align the **background-color** of the `quick-open` to match that of vscode.
Currently, on dark themes the shadow is unnoticeable. 

The **background-color** should be the same as docked widgets in the sidebar, so the value should be
```css
--theia-layout-color1
```

Reply:
I created an issue for it https://github.com/theia-ide/theia/issues/5735, but I believe it should be addressed with this PR, you can add a commit on top of your current one to include it :)

Reply:
Sure, I will do it ;)